libs/inspecjs/src/nist.ts

```diff
@@ -2,9 +2,10 @@ import {ControlStatus} from './compat_wrappers';
 import {ALL_NIST_CONTROL_NUMBERS, ALL_NIST_FAMILIES} from './raw_nist';
 
 // Regexes.
-const NIST_FAMILY_RE = /^[A-Z]{2}$/;
+// Matches against only currently existing NIST Control families instead of allowing any two adjacent uppercase letters
+const NIST_FAMILY_RE = /^(?:A[CPRTU]|C[AMP]|D[IM]|I[APR]|M[AP]|P[ELMS]|RA|S[ACEI]|TR|U[LM])$/;
```

Comment:
would appreciate you pleasing the linter on my behalf and also regenerating the samples

Reply:
Done! The linting issues have been addressed and samples regenerated.